src/cfg.rs

```diff
@@ -262,6 +284,15 @@ fn parse_cfg_raw(
         })
         .collect::<Vec<_>>();
 
+    let mut layer_info = vec![];
```

Comment:
I would prefer a direct assignment to layer_info by using map and collect rather than making it mut and pushing.

```
// note: not checked for correctness
let layer_info: Vec<LayerInfo> = layer_names
    .iter()
    .zip(layer_strings)
    .map(|(name, cfg_text)| LayerInfo {name, cfg_text})
    .collect();
```

Reply:
Implemented this change